Macros/DMPED_pipeline_read_update_file.sas

```diff
@@ -0,0 +1,375 @@
+	DCHFA_Units_51_60_AMI = "Number of DCHFA units at 51-60% of AMI"
+	DCHFA_Units_61_80_AMI = "Number of DCHFA units at 61-80% of AMI"
+	DCHFA_Units_Affordable = "Total number of DCHFA affordable units"
+	DHCD_DFD_0_30_AMI = "Number of DHCD-DFD units at 0-30% of AMI"
+	DHCD_DFD_31_40_AMI = "Number of DHCD-DFD units at 31-40% of AMI"
+	DHCD_DFD_41_50_AMI = "Number of DHCD-DFD units at 41-50% of AMI"
+	DHCD_DFD_51_60_AMI = "Number of DHCD-DFD units at 51-60% of AMI"
+	DHCD_DFD_61_80_AMI = "Number of DHCD-DFD units at 61-80% of AMI"
+	DHCD_DFD_allsubsidy = "Total number of DHCD-DFD subsidized units"
+	DHCD_DFD_pubhousing = "Total number DHCD-DFD public housing units"
+	DHCD_DFD_total = "Total number of DHCD-DFD affordable units"
+	DMPED_PPD_0_30_AMI ="Number of DMPED-PPD units at 0-30% of AMI"
+	DMPED_PPD_31_50_AMI ="Number of DMPED-PPD units at 31-50% of AMI"
+	DMPED_PPD_51_60_AMI ="Number of DMPED-PPD units at 51-60% of AMI"
+	DMPED_PPD_61_80_AMI ="Number of DMPED-PPD units at 61-80% of AMI"
+	HPTF_Affordable_Units = "Number of Housing Production Trust Fund affordable units"
+	IZ_Database_total = "Total number of IZ units"
+	IZ_Database_0_50_AMI = "Number of IZ units at 0-50% of AMI"
+	IZ_Database_51_60_AMI = "Number of IZ units at 51-60% of AMI"
+	IZ_Database_61_80_AMI = "Number of IZ units at 61-80% of AMI"
+	PUD_Affordable_Units = "Total number of PUD affordable units"
+	PUD_Units_0_30_MFI = "Number of PUD units at 0-30% of MFI"
+	PUD_Units_31_60_MFI = "Number of PUD units at 31-60% of MFI"
+	PUD_Units_61_80_MFI = "Number of PUD units at 61-80% of MFI"
+	PUD_Units_81_MFI = "Number of PUD units over 81% of MFI"
+	TOPA = "TOPA Project"
+	DHCD_0_30_AMI = "Number of DHCD units at 0-30% of AMI"
+	DHCD_31_50_AMI = "Number of DHCD units at 31-50% of AMI"
+	DHCD_51_60_AMI = "Number of DHCD units at 51-60% of AMI"
+	DHCD_61_80_AMI = "Number of DHCD-DFD units at 61-80% of AMI";
+
+  run;
+
+  data 
+    IZ_Units_&year._&month (label="&ds_label, DC") ;
+ infile "&folder\raw\Housing Pipeline\iz_units_&filedate_fmt..csv" dsd stopover lrecl=2000 firstobs=2;
+input
+	Project :$80.
+	Construction_Status :$40.
+	Project_Address :$80.
+	IZ_Unit_Num :$10.
+	Tenure :$10.
+	Bedrooms :1.
+	AMI :$10.
+	Estimated_Availability_Date :$12.
+	Bedrooms_iz_def : 2.
+	Unit_Total :3.;
+	    retain Extract_date;
+       label
+	project = "Project name"
+	construction_status = "Construction status"
+	project_address = "Project address"
+	iz_unit_num = "Unit Number"
+	tenure = "Rental or owner"
+	bedrooms = "Number of bedrooms"
+	AMI = "AMI level of unit"
+	estimated_availability_date = "Date of availability"
+	bedrooms_iz_def = "Number of bedrooms using IZ definition"
+	unit_total = "Total number of IZ units in building"
+run;
+
+  ** Finalize data sets **;
+
+    %Finalize_data_set( 
+      /** Finalize data set parameters **/
+      data=iz_&year._&month,
+      out=iz_&year._&month,
+      outlib=DMPED,
+      label="&ds_label, %upcase(&v)",
```

Comment:
Not sure what the &v was supposed to be but that macro variable has not been defined.